script.audio.profiles/default.py

```diff
@@ -6,187 +6,149 @@
 import xbmcvfs
 import json
 import os
+import sys
 
-__addon__ = xbmcaddon.Addon()
-__addon_id__ = __addon__.getAddonInfo('id')
-__addonpath__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
-__datapath__ = xbmc.translatePath(os.path.join('special://profile/addon_data/', __addon_id__)).replace('\\', '/') + '/'
-__path_img__ = __addonpath__ + '/images/'
-__lang__ = __addon__.getLocalizedString
+__addon__               = xbmcaddon.Addon()
```

Comment:
variable names with double underscore prefix/suffix should be avoided, that´'s reserved for python internals.